src/interface/IWithdrawalQueueManager.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: BSD 3-Clause License
+pragma solidity ^0.8.24;
+
+interface IWithdrawalQueueManager {
```

Comment:
good solution now to use interface as opposed to submodules due to remappings.txt hell